workflow/rules/qc_3prime.smk

```diff
@@ -58,6 +58,7 @@ if is_3prime_experiment and config["experiment"]["3-prime-rna-seq"]["plot-qc"] !
                 "results/plots/QC/3prime-ind-QC-plot.{ind_transcripts}.html",
                 category="QC",
                 caption="../report/plot-3prime-QC-histogram.rst",
+                labels={"QC-plot": "{ind_transcripts}-QC-plot"},
```

Comment:
This QC-plot naming is still super-unspecific, I mentioned that quite often now.
```
category="QC",
subcategory="global",
labels={"plot": "3-prime read positioning", "subset": "{ind_transcripts}"}
```

Reply:
Noted. Done